addons/survey/models/survey_question.py

```diff
@@ -472,12 +472,22 @@ def _validate_date(self, answer):
         return {}
```

Comment:
Commit message:

> It also counts a comment as an answer if a comment is set
as a valid answer.

That was the case before though, right?

From what I understand, what you changed here is that it no longer allows having a comment as an answer **in addition to** a regular answer.